asistente_ladm_col/core/reports/base_report_generator.py

```diff
@@ -186,6 +188,9 @@ def validate_dependencies(self):
         if not self.check_java_dependency():
             return False
 
+        if not self.spatial_layers_is_valid():
```

Comment:
Replace `spatial_layers_is_valid` by `spatial_layers_are_valid`.

Reply:
changes made in commit 604cc6e